CI/AzureStaticWeb/src/index.js

```diff
@@ -0,0 +1,189 @@
+import React from "react";
+import { render } from "react-dom";
+import _ from "lodash";
+import "./index.css";
+
+// This sample uses React Table
+// Check it out at https://react-table.js.org/#/story/readme
+import ReactTable from "react-table";
+import "react-table/react-table.css";
+
+// Import Azure Storage Blob SDK
+import { Aborter, ServiceURL, ContainerURL, StorageURL, AnonymousCredential } from "@azure/storage-blob";
+
+// Account name, and the container to list from
+const account = 'featurizerslibrarybuild'
+const container = 'archive'
+
+class App extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            data: [],
+            pages: 2,
+            markers: [],
+            loading: true,
+            prefix: ""
+        };
+        this.fetchData = this.listBlobs.bind(this);
+    }
+
+    listBlobs(state, instance) {
+        // this lists Blobs in pages defined in state.pageSize
+        this.setState({ loading: true });
+
+        // Use AnonymousCredential since $web container is made a 'public container'
+        // and does not require authorization
+        const anonymousCredential = new AnonymousCredential();
+        const pipeline = StorageURL.newPipeline(anonymousCredential);
+
+        const serviceURL = new ServiceURL(
+            `https://${account}.blob.core.windows.net`,
+            pipeline
+        );
+
+        // If you are using a SAS token, simply append to ContainerURL here.
+        // We will use anonymous access hence no SAS token
+        const containerName = container //+ `?st=2018-11-06T06%3A15%3A24Z&se=2019-11-07T06%3A15%3A00Z&sp=rl&sv=2018-03-28&sr=c&sig=4vCT7aInDWRiypkuYlezN8dos0K2h2DvQ0pnNkMJSFs%3D`;
```

Comment:
We shouldn't leave a SAS token in here with a comment.

Reply:
Yeah you are right!